jade/page-contents/range_content.html

```diff
@@ -44,6 +44,11 @@ <h3 class="header">HTML5 Range</h3>
     &lt;/p>
   &lt;/form>
         </code></pre>
+        <p>Initialize with javascript to display the value above the slider when dragging</p>
+        <pre><code class="language-javascript">
+  var array_of_dom_elements = document.querySelectorAll("input[type=range]");
```

Comment:
`Range.init` doesn't take a jQuery object. `Range.init($('input[type=range]')[0])` works but seems hackish. Since jQuery isn't a dependency anymore I feel it shouldn't get special treatment.

Reply:
I see what you mean. I was thinking of this though: `$("input[type=range]").range()`

(though I'm only *assuming* that's the jQuery way from looking at the other plugins)